hassio/src/components/hassio-ansi-to-html.ts

```diff
@@ -0,0 +1,259 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+interface State {
+  bold: boolean;
+  italic: boolean;
+  underline: boolean;
+  strikethrough: boolean;
+  foregroundColor: null | string;
+  backgroundColor: null | string;
+}
+
+export const ANSI_HTML_STYLE = css`
```

Comment:
Is this still needed as an exported const? Or can we directly add it to styles?

Reply:
We can add it directly! 🎉